website/content/plugins/devices/nvidia.mdx

```diff
@@ -12,7 +12,8 @@ Use the NVIDIA device plugin to expose NVIDIA graphical processing units (GPUs)
 to Nomad. The driver automatically supports [Multi-Instance GPU (MIG)][mig].
 
 The NVIDIA device plugin uses [NVML] bindings to get data regarding available
-NVIDIA devices and then exposes them via [Fingerprint RPC]. The plugin detects
+NVIDIA devices (for a list of supported cards, consult [NVIDIA's documentation](https://docs.nvidia.com/deploy/nvml-api/nvml-api-reference.html#nvml-api-reference))
+and then exposes them via [Fingerprint RPC]. The plugin detects
```

Comment:
```suggestion
NVIDIA devices and then exposes them via [Fingerprint RPC]. Consult [NVIDIA's
documentation](https://docs.nvidia.com/deploy/nvml-api/nvml-api-reference.html#nvml-api-reference)
for a list of supported cards. The plugin detects
```

I moved the NVIDIA docs to its own sentence so we aren't interrupting the "uses and exposes" sentence. Also, we like to avoid putting content inside parentheses because people tend to not read that content.